sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_connection_async.py

```diff
@@ -237,7 +253,7 @@ async def _read_frame(self, wait: Union[bool, int, float] = True, **kwargs) -> b
         """
         timeout: Optional[Union[int, float]] = None
         if wait is False:
-            timeout = 1  # TODO: What should this default be?
+            timeout = self._socket_timeout
```

Comment:
since this was previously 1, just to make sure changing to 0.2 doesn't affect anything on the async side, I think we might need to stress test?

Reply:
yep we will have to stress test. Im quite positive the end frame fix will also yield some positive results during stress

At the protocol level, this has already been tested under SB as we are bringing in the changes from there.

Reply:
discussed offline that defaults at the pyamqp level are fine. We should pass in other values that suit the services at the service layer